tests/load/sources/rest_api/test_rest_api_source.py

```diff
@@ -0,0 +1,128 @@
+from typing import Any
+import dlt
+import pytest
+from dlt.sources.rest_api.typing import RESTAPIConfig
+from dlt.sources.helpers.rest_client.paginators import SinglePagePaginator
+
+from dlt.sources.rest_api import rest_api_source
+from tests.pipeline.utils import assert_load_info, load_table_counts
+from tests.load.utils import (
+    destinations_configs,
+    DestinationTestConfiguration,
+)
+
+
+def _make_pipeline(destination_name: str):
+    return dlt.pipeline(
+        pipeline_name="rest_api",
+        destination=destination_name,
+        dataset_name="rest_api_data",
+        full_refresh=True,
+    )
+
+
+@pytest.mark.parametrize(
+    "destination_config",
+    destinations_configs(default_sql_configs=True),
```

Comment:
also add local filesystem configs

Reply:
Done in d47cf64866100b36be1218cf33dc03a285bac12f